addons/l10n_es_edi_tbai/data/template_invoice.xml

```diff
@@ -184,7 +184,7 @@
             <NoSujeta t-if="desglose.get('NoSujeta')">
                 <t t-set="no_sujeta" t-value="desglose['NoSujeta']"/>
                 <DetalleNoSujeta>
-                    <Causa>RL</Causa>
+                    <Causa t-out="nosujeto_causa"/>
                     <!-- NOTE: Causa should be 
                         'OT' if 'the' ClaveRegimenIvaOpTrascendencia == 10
                         'RL' if 'some' ClaveRegimenIvaOpTrascendencia == 08
```

Comment:
As well it seems logical to do OT when ImportePorArticulos > 0 and RL when ReglasLocalizacion > 0, but we can merge this already